source/fundamentals/utf8-validation.txt

```diff
@@ -0,0 +1,116 @@
+.. _nodejs-utf-8-validation:
+
+================
+UTF-8 Validation
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to enable or disable the {+driver-short+}'s
+**UTF-8** validation feature. **UTF-8** is a character encoding specification
+that ensures compatibility and consistent presentation across most operating
+systems, applications, and language character sets. By default, the driver
+checks documents for any characters that are not encoded in a valid UTF-8
+format when it transfers data between your application and MongoDB.
+
+If you enable validation, the driver substitutes invalid UTF-8 characters
+with valid UTF-8 ones when transferring data between your application and
+MongoDB. The validation adds processing overhead since it needs to verify the
+data and to make substitutions when necessary.
```

Comment:
[comment] Minor, but this reads like an extra "to" to me since you already say "needs to verify..." 
```suggestion
data and make substitutions when necessary.
```